src/seahawk_rov/package.xml

```diff
@@ -7,11 +7,21 @@
   <maintainer email="michael.matera@cabrillo.edu">Mike Matera</maintainer>
   <license>TODO: License declaration</license>
 
-  <exec_depend>example_interfaces</exec_depend>
-  <exec_depend>rclpy</exec_depend>
-
-  <buildtool_depend>ament_python</buildtool_depend>
+  <!-- Explicitly depend upon setuptools. In the event that it's missing, `rosdep` will fetch it for us before we build -->
+  <build_depend>python3-setuptools</build_depend>
 
+  <exec_depend>rclpy</exec_depend>
+  <exec_depend>std_msgs</exec_depend>
+  <exec_depend>sensor_msgs_py</exec_depend> <!-- TODO: Check if we need one, or both of "sensor_msgs{,_py}". It's the only one that has a _py variant. Strange :l-->
+  
+  <exec_depend>python3</exec_depend>
+  <exec_depend>python-rpi.gpio-pip</exec_depend>
+  <exec_depend>python3-adafruit-blinka-pip</exec_depend>
```

Comment:
is this an apt package name or is this some kind of rosdep meta package thing?

Reply:
Rosdep doesn't have it's own package sources the same way that apt or pip do. Instead, it reads a list of mappings between rosdep "keys" (the names we see here) and the names of these other tools. There's some notes about it in the doc/rosdep_nodes.md file (and links to the official docs, ofc).